azurestack/data_source_platform_image_test.go

```diff
@@ -0,0 +1,40 @@
+package azurestack
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataAZURESTACKPlatformImage_basic(t *testing.T) {
```

Comment:
it'd be good to re-order/update the casing here so that this is caught by the default prefix:

```suggestion
func TestAccAzureStackDataPlatformImage_basic(t *testing.T) {
```

Reply:
Whoops, find replace gone aray